frontend/www/js/omegaup/components/problem/Details.test.ts

```diff
@@ -123,7 +123,26 @@ describe('Details.vue', () => {
     qualityHistogram: '[0,1,2,3,4]',
   };
 
-  it('Should handle no nomination payload', () => {
+  const runs = [
```

Comment:
```suggestion
  const runs = <types.Run[]>[
```

así abajo ya no hay necesidad de hacer el cast.

Reply:
Listo